server/channels/api4/webhook_test.go

```diff
@@ -1235,20 +1252,23 @@ func TestUpdateOutgoingHook(t *testing.T) {
 		createdHook.ChannelId = ""
 		createdHook.TriggerWords = nil
 
-		_, resp, err := client.UpdateOutgoingWebhook(context.Background(), createdHook)
+		var resp *model.Response
+		_, resp, err = client.UpdateOutgoingWebhook(context.Background(), createdHook)
 		require.Error(t, err)
 		CheckInternalErrorStatus(t, resp)
 	}, "UpdateToBlankTriggerWordAndChannel")
 
 	team := th.CreateTeamWithClient(th.Client)
 	user := th.CreateUserWithClient(th.Client)
 	th.LinkUserToTeam(user, team)
-	th.Client.Logout(context.Background())
-	th.Client.Login(context.Background(), user.Id, user.Password)
+	_, err = th.Client.Logout(context.Background())
+	require.NoError(t, err)
+	_, _, err = th.Client.Login(context.Background(), user.Username, user.Password)
+	require.Error(t, err)
```

Comment:
This no longer errors :wink: 
```suggestion
	require.NoError(t, err)
```

Reply:
Sorry forgot about this.